atlas/responder/.http

```diff
@@ -0,0 +1,77 @@
+###
```

Comment:
I though responder should only have GRPC endpoints? Or did you add gateway to help with testing and debug?